rbkit-lib/cpu/cpuprof.cpp

```diff
@@ -0,0 +1,53 @@
+#include <QCoreApplication>
+#include <QtCore/QJsonDocument>
+#include <QtCore/QJsonObject>
+#include <QtCore/QJsonArray>
+#include <QFile>
+#include <QTextStream>
+#include <QDebug>
+#include <typeinfo>
+#include "cpustorage.h"
+#include "cpumapping.h"
+#include <iostream>
+#include <string.h>
+#include <QDebug>
+#include <QList>
+
+#include "cpuprof.h"
+
+
+static storage *store = new storage();
+
+void CpuProf::parseFrames(QMap<int, QVariant> *frames) {
```

Comment:
dont use a pointer to frames. directly pass QMap. as i explained you earlier copy-on-write feature from Qt is quite efficient on Qt core structures.